src/applications/personalization/profile/tests/e2e/contact-information/focus-management.cypress.spec.js

```diff
@@ -39,9 +32,11 @@ describe('Contact info fields', () => {
     cy.injectAxe();
 
     cy.findByRole('button', { name: /edit mailing address/i }).click();
```

Comment:
should we also update this line so that it follows what you are doing below? 
EX: `cy.get('va-button[text="Edit mailing address"]').click();`

Reply:
I also have the urge to update everything, but with 449 lines of code changes for this PR already, I try to only change code that needs to be changed. I've gotten the side-eye for my 1000+ lines of code change PRs 😭